virttest/env_process.py

```diff
@@ -1134,6 +1134,15 @@ def postprocess(test, params, env):
     # recover the changes done to kernel params in postprocess
     kernel_extra_params_add = params.get("kernel_extra_params_add", "")
     kernel_extra_params_remove = params.get("kernel_extra_params_remove", "")
+
+    if params.get("enable_guest_iommu"):
+        enable_guest_iommu = params.get("enable_guest_iommu")
+        if enable_guest_iommu == "yes":
+            kernel_extra_params_add += " intel_iommu=on"
```

Comment:
If you wanna clean up, should be `kernel_extra_params_remove`? And what's the use case of this action?

Reply:
'kernel_extra_params_add' is the correct usage, please refer to Line 1151.
The use case is described in bug.
Thanks!

Reply:
Okay, the code looks okay to me, just please demonstrate same content in commit log. Thanks.

Reply:
Updated!